src/api/app/views/webui/request/_action_text.html.haml

```diff
@@ -1,31 +1,37 @@
-- details = action.commit_details
-- if details
-  - if details.key?('comment')
-    %p.mt-0.mb-0.fw-normal
-      %span.me-2 ##{action_index + 1}
-      = details['comment'].truncate(50)
-    %p.mb-0.mt-0.small
-      = action.name
-  - else
-    %p.mt-0.mb-0.fw-normal
-      %span.me-2 ##{action_index + 1}
-      = action.name
-  %span.float-end.ms-5
-    = details['rev']
-- else
-  %p.mt-0.mb-0.fw-normal
-    %span.me-2 ##{action_index + 1}
-    = action.name
--# add_role, delete and set_bugowner are actions on the target repo, so they don't need to mention where they are coming from
-- unless %w[add_role delete set_bugowner].include?(action.type)
-  %p.mb-0.mt-0.small
-    from
-    = action.source_project
-    - if action.source_package
-      \/
-      = action.source_package
-- if details
-  %p.mb-0.mt-0.small
-    by
-    = details['user']
-    #{ApplicationController.helpers.time_ago_in_words(details['time'].to_i)} ago
+.d-flex.align-items-center
+  - if User.session
+    .ms-1.me-3{ id: "action-seen-by-user-wrapper-in-select-#{action.id}" }
+      = render ActionSeenByUserComponent.new(action: action, user: User.session!, render_only: true)
```

Comment:
This would call `User.session/session!` only once:

```suggestion
  - if (user = User.session)
    .ms-1.me-3{ id: "action-seen-by-user-wrapper-in-select-#{action.id}" }
      = render ActionSeenByUserComponent.new(action: action, user: user, render_only: true)
```